### PR TITLE
Fix the api-reference pages in gh-pages, and update the _tool/

### PR DESCRIPTION
**Pull request by caesarxuchao**
Now the tool in _tool/ strips the <!DOCTYPE> and <html> tags in definitions.html and operations.html, and move them to _includes/.

Also pushed a third commit to remove the underlines when rendering hyperlinks in these two pages. It looks more consistent with other pages now.

You can check the effects here: http://caesarxuchao.github.io/v1.0/docs/api-reference/operations.html

replace #11850 #11851

@thockin @RichieEscarez @nikhiljindal

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@RichieEscarez, it seems the width limit of these two pages are removed, which is what we want.

File: _tools/release_docs/api-reference-process.go [hunk 1]
```diff
@@ -66,44 +42,59 @@ func addHeader(filename string) error {
...
 	if err != nil {
 		return err
 	}
-	newContents := strings.Replace(string(file), "h2 { font-size: 2.3125em; }", "h2 { font-size: 2.3125em; text-align: left;}", 1)
-	newContents = strings.Replace(newContents, "h4 { font-size: 1.4375em; } }", "h4 { font-size: 1.4375em; text-align: left;} }", 1)
-	newContents = strings.Replace(newContents, "<h2 id=\"_paths\">Paths</h2>", "<h2 id=\"_paths\">Operations</h2>", 1)
+	//TODO: This really should be fixed when generating the original html files
+	newContents := strings.Replace(string(file), "<h2 id=\"_paths\">Paths</h2>", "<h2 id=\"_paths\">Operations</h2>", 1)
+	//Adapt the color of headings to k8s style
+	newContents = strings.Replace(newContents, "color: #ba3925", "color: #428BCB", -1)
```
Comment (thockin) [inline _tools/release_docs/api-reference-process.go]: Medium-term this should all come from style, yes?

**Comment by thockin** [comment]
LGTM, merging to unbreak it.

**Comment by thockin** [comment]
I still get an errant - underneath the top-bar items when viewing the api-reference files

File: _tools/release_docs/api-reference-process.go (hunk 1 above)
Comment (caesarxuchao) [inline _tools/release_docs/api-reference-process.go]: The color is defined in the style of hmtl files in the master branch. We need some of the style to keep the table format. I think we should look for a way to overwrite these using _layouts/ and _scss/.

**Comment by caesarxuchao** [comment]
ah, I failed to fix it this morning, but I think I just found what caused it. I will submit a patch tomorrow.

**Comment by thockin** [comment]
Also the font and font size are different for these 2 docs.

**Comment by thockin** [comment]
Also the search bar is gone on these 2 pages.

**Comment by thockin** [comment]
Getting rid of the &nbsp; in _includes/dochead.html and
_includes/header.html underline class makes that mark go away

**Comment by thockin** [comment]
I'll let you deal with it tomorrow.  The main urgent thing is fixed, thanks.

**Pull request merged**